x/connectivity/errors_unix.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2023 Jigsaw Operations LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build unix
+
+package connectivity
+
+import (
+	"syscall"
+
+	"golang.org/x/sys/unix"
+)
+
+func errnoName(errno syscall.Errno) string {
+	return unix.ErrnoName(errno)
```

Comment:
Shall we return the raw error code if the name is not found to align with the behavior in Windows code, cuz [`ErrnoName` will return `""`](https://cs.opensource.google/go/x/sys/+/refs/tags/v0.8.0:unix/syscall_unix.go;l=54-62) if the error name is not found?

```suggestion
	if name := unix.ErrnoName(errno); len(name) > 0 {
		return name
	}
	return fmt.Sprintf("Error %d (0x%x)", int(errno), int(errno))
```

Reply:
Good point. Fixed!